products/storage/terraform.yaml

```diff
@@ -102,6 +102,28 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
   Object: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  HmacKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{service_account_email}}
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      post_create: templates/terraform/post_create/hmac_secret.go.erb
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "storage_hmac_key"
+        primary_resource_id: "key"
+        vars:
+          service_account_email: "my-service-account@my-project.iam.gserviceaccount.com"
+    id_format: "projects/{{project}}/accessId/{{access_id}}"
+    properties:
+      metadata: !ruby/object:Overrides::Terraform::PropertyOverride
+        flatten_object: true
+      metadata.id: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      secret: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Can you add a `docs` warning that `secret` will be unavailable for users that import an existing key, and that it will only be populated for keys created in TF? Here's an example: https://github.com/GoogleCloudPlatform/magic-modules/blob/e6a4cf21eca53b6edc422509a744497f4a61e03b/products/compute/terraform.yaml#L172-L177